dokka-integration-tests/settings.gradle.kts

```diff
@@ -10,8 +10,8 @@ pluginManagement {
     includeBuild("../build-logic")
 
     repositories {
-        gradlePluginPortal()
         mavenCentral()
+        gradlePluginPortal()
```

Comment:
Minor unrelated change. Maven Central is generally more stable than the Gradle Plugin portal, so prioritising Maven Central helps with build stability.